fields/types/list/ListField.js

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable react/jsx-no-bind */
+
+import assign from 'object-assign';
+import React from 'react';
+import Field from '../Field';
+import Domify from 'react-domify';
+
+import { Fields } from 'FieldTypes';
+import { Button, GlyphButton } from '../../../admin/client/App/elemental';
+import InvalidFieldType from '../../../admin/client/App/shared/InvalidFieldType';
+
+function generateId () {
+	return Math.random().toString(36).substr(2, 9);
```

Comment:
This is just here to give React unique keys for multiple new items in an array.

It's not going to be submitted, and gets reset when you save, so you'd have to be _incredibly_ lucky to get two random keys in that much namespace to conflict. Like, impossible.

Having said that - we could just use an incrementing integer. 😁